src/main/java/edu/umich/its/cpm/MigrationController.java

```diff
@@ -2131,14 +2131,18 @@ private JSONObject googleGlobalFailureReport(JSONObject statusObj, JSONObject de
 				//String userEmail = Utils.getUserEmailFromUserId(userEid);
 				String userEmail = getUserEmailFromUserId(userEid);
 
-				BoxUtils.addCollaboration(
-						env.getProperty(Utils.BOX_ADMIN_ACCOUNT_ID),
-						userEmail, userRole, boxFolderId,
-						boxAdminClientId, boxAdminClientSecret, uRepository);
-				// add user email to the owner list
-				if (addUserEmail(siteId, userRole)) 
+				// exclude the temporary added site admin account
+				if (!userEmail.startsWith(env.getProperty(Utils.BOX_ADMIN_ACCOUNT_ID)))
```

Comment:
How about this (to avoid nesting) ?
if (userEmail.startsWith(env.getProperty(Utils.BOX_ADMIN_ACCOUNT_ID))) {
continue;
}